### ISSUE TITLE
German Setup for Magento 1.9 and up ??

### ISSUE THREAD

**Issue by sr-design** [firegento/firegento-germansetup#222]
Hi,

is there an version for magenta 1.9 and up ?

greets

Stefan

**Comment by riconeitzel** [comment]
Yes, but it's transfered to https://github.com/firegento/firegento-magesetup which is the same code base + more new and awesome stuff!

GermanSetup's Development has been discontinued.

Just switch the extensions - that's all.

Rico
